### PR TITLE
Fix "If, lines" codestyle

### PR DESCRIPTION
**Pull request by jejikeh**

### CODE REVIEW

File: src/Versity.Users/Core/Application/RequestHandlers/Auth/Commands/LoginVersityUser/LoginVersityUserCommandHandler.cs [hunk 1]
```diff
@@ -21,12 +21,10 @@ public LoginVersityUserCommandHandler(IAuthTokenGeneratorService tokenGeneratorS
...
+        if (versityUser is null || !await _versityUsersRepository.CheckPasswordAsync(versityUser, request.Password)) 
+        { 
             throw new IncorrectEmailOrPasswordExceptionWithStatusCode();
-
-        if (!await _versityUsersRepository.CheckPasswordAsync(versityUser, request.Password))
-            throw new IncorrectEmailOrPasswordExceptionWithStatusCode();
-
+        }
         var userRoles = await _versityUsersRepository.GetRolesAsync(versityUser);
         return _tokenGeneratorService.GenerateToken(versityUser.Id, versityUser.NormalizedEmail, userRoles);
```
Comment (KatsiarynaAnashkevich) [inline src/Versity.Users/Core/Application/RequestHandlers/Auth/Commands/LoginVersityUser/LoginVersityUserCommandHandler.cs]: Empty line before return) Try to remember about it

File: src/Versity.Users/Core/Application/RequestHandlers/Auth/Commands/RegisterVersityUser/RegisterVersityUserCommandHandler.cs [hunk 2]
```diff
@@ -29,7 +30,6 @@ public async Task<IdentityResult> Handle(RegisterVersityUserCommand request, Can
             LastName = request.LastName,
             UserName = $"{request.FirstName} {request.LastName}",
         };
-
         var result = await _versityUsersRepository.CreateUserAsync(versityUser, request.Password);
         await _versityUsersRepository.SetUserRoleAsync(versityUser, VersityRole.Member);
         return result;
```
Comment (KatsiarynaAnashkevich) [inline src/Versity.Users/Core/Application/RequestHandlers/Auth/Commands/RegisterVersityUser/RegisterVersityUserCommandHandler.cs:39]: same

File: src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs [hunk 3]
```diff
@@ -40,16 +42,18 @@ public static async Task<IServiceProvider> EnsureRolesExists(this IServiceProvid
         var anyRoleWasAdded = false;
         foreach (var role in roles)
```
Comment (KatsiarynaAnashkevich) [inline src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs]: Not the best way to add roles

**Review by KatsiarynaAnashkevich — COMMENTED** [review]
Check empty line before return

File: src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs (hunk 3 above)
Comment (jejikeh) [reply src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs]: Actually, i think i even dont need to manualy add roles here, bc ASP.Core Identity already comes with roles, such as "Administrator" or "Member".

Comment (jejikeh) [reply src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs]: And I do not know how to do it differently :\

Comment (KatsiarynaAnashkevich) [reply src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs]: You can create something like data seeder and here add neccessary for your app roles and etc

Comment (jejikeh) [reply src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs]: I made a data seeder for [roles](https://github.com/jejikeh/versity/pull/17/files#diff-0b0e7c1a0378497d9a445905ecd9d18a4b9a7b4d4b53ad992af976a712df5130), [admin](https://github.com/jejikeh/versity/pull/17/files#diff-364cc66a27706bc78600c4f821c48f3e0a60a30a3fd9bc60d72e2eef9e169646) and [userRoles](https://github.com/jejikeh/versity/pull/17/files#diff-20cdd38d5bffa7b157beaaaaf089d0213c1c6e4ca034f0008f987ae2bd4e65c6)

**Pull request merged**